target/linux/ipq40xx/base-files/lib/upgrade/linksys.sh

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (C) 2014-2015 OpenWrt.org
+#
+
+linksys_get_target_firmware() {
+	cur_boot_part=`/usr/sbin/fw_printenv -n boot_part`
```

Comment:
The backticks/gravemarks have been deprecated in favor of $()

Though, this is fine as is.